app/assets/javascripts/govuk.js

```diff
@@ -8,6 +8,10 @@ $(document).ready(function () {
 
   // Clear cookie set by govuk_template.
   GOVUK.cookie('seen_cookie_message', null);
+  document.cookie = '_ga=;expires=' + new Date() + ';domain=' + window.location.hostname + ';path=/'
+  document.cookie = '_gat=;expires=' + new Date() + ';domain=' + window.location.hostname + ';path=/'
+  document.cookie = '_gid=;expires=' + new Date() + ';domain=' + window.location.hostname + ';path=/'
+  document.cookie = 'dgu_beta_banner_dismissed=;expires=' + new Date() + ';domain=' + window.location.hostname + ';path=/'
```

Comment:
I guess this removes the cookie by setting it to expire now (`new Date()`)... that API is odd.

Reply:
Yes, I had a chat with @vanitabarrett about this. I'm not sure if/how we can do this differently. Do you have suggestions?